ajax/libs/kube/package.json

```diff
@@ -16,10 +16,11 @@
   "autoupdate": {
     "source": "git",
     "target": "git://github.com/imperavi/kube.git",
-    "basePath": "",
+    "basePath": "dist",
     "files": [
       "css/*",
-      "js/*"
+      "js/*",
+      "!(scss)/*"
```

Comment:
I prefer to use "exclude" pattern, at least we can know that there are something changed, unless the author says "I can keep my eyes on it and send a PR to CDNJS when the structure changed".